versioned_docs/version-8.6/guides/migrating-from-camunda-7/conceptual-differences.md

```diff
@@ -48,6 +47,8 @@ Camunda 8 uses [Friendly-Enough Expression Language (FEEL)](/components/modeler/
 
 Most expressions can be converted (see [this code in the diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/blob/main/backend-diagram-converter/core/src/main/java/org/camunda/community/migration/converter/expression/ExpressionTransformer.java) as a starting point), some might need to be completely rewritten, and some might require an additional service task to prepare necessary data (which may have been calculated on the fly when using Camunda 7).
 
+// TODO extensive docs for the diagram converter
```

Comment:
I think I would rather push that to the code conversion section - not here. But we can leave the todo for now and see when we write the content where to push it and how to link